gulpfile.js

```diff
@@ -15,13 +16,21 @@ const gulp = require('gulp'),
     .demandOption(['c', 'd'])
     .argv;
 
+var appSrc = 'src';
+var libraryDist = 'dist';
+
 function copyToDest(srcArr) {
   return gulp.src(srcArr)
     .pipe(gulp.dest(function (file) {
       return libraryDist + file.base.slice(__dirname.length); // save directly to dist
     }));
 }
 
+//Less compilation and minifiction - adopted from sass compilation, needs work
+gulp.task('transpile-less', function () {
+  return transpileLESS(appSrc + '/app/**/*.less');
```

Comment:
I suspect we want to revert theses changes as webpack is now performing the less conversion.